doctordroid/README.md

```diff
@@ -0,0 +1,35 @@
+# Doctor Droid
+
+## Overview
+
+Doctor Droid is an observability analytics platform that analyzes alerts history and fetches metrics from upstream and downstream services. Use the integration to:
+- Decide better thresholds for alerts
+- Identify alerting coverage for your services
+- Enrich alerts for faster diagnosis
+
+## Setup
+
+### Installation
+Visit [Doctor Droid][1] to sign up for free. Once registered, visit the [integrations page][2]  and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Doctor Droid access to your API Keys to query APM & Infra metrics for you.
```

Comment:
Please prompt the user to instead click Install on the Datadog integration tile, then click "Connect Accounts" to kick off the OAuth flow. At that point they will be brought to your onboarding_URL.